requirements.txt

```diff
@@ -5,6 +5,7 @@
 -r e2e_tests/tests/requirements.txt
 -r bindings/requirements.txt
 
+requests<2.32.0 # https://github.com/docker/docker-py/issues/3256
```

Comment:
I would move this to the files in `harness/tests/requirements` since this is a dependency of our CLI/SDK deploy code.